students/hyerim/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from users.views import UserView
+
+urlpatterns = [
+    path('/user_join', UserView.as_view()),
```

Comment:
```suggestion
    path('/signup', UserView.as_view()),
```
최종적인 엔드포인트는 users/user_join 이 되겠네요. 중복을 피하고 의미를 명확히 하기 위해 위와 같이 수정해주세요.